tests/test_wcon.py

```diff
@@ -69,43 +25,152 @@ def test_wcon2df():
         assert col == col.upper()
 
 
-def test_str2df():
-    """Test dataframe extraction from strings"""
+def test_wconhist():
+    """Test WCONHIST parsing and column names"""
     wconstr = """
 WCONHIST
   'FOO' 0 1 /
  /
 """
```

Comment:
Blir det penere kode når teksten i wconstr har et ekstra innrykk?

Reply:
Perhaps,  but that is also a different Eclipse deck with whitespace preceding the Eclipse keyword at line start (which is not common in Eclipse files)